galley/pkg/server/components/validation.go

```diff
@@ -27,10 +27,12 @@ func NewValidation(kubeConfig string, params *validation.WebhookParameters, live
 	return process.ComponentFromFns(
 		// start
 		func() error {
+			webhookServerReady := make(chan struct{})
+			stopCh := make(chan struct{})
 			if params.EnableValidation {
-				webhookServerReady := make(chan struct{})
-				stopCh := make(chan struct{})
 				go validation.RunValidation(webhookServerReady, stopCh, params, kubeConfig, liveness, readiness)
+			}
+			if !params.DisableReconcileWebhookConfiguration {
```

Comment:
@ayj @clyang82 Should this be inverted to "EnableReconcileWebhookConfiguration" so it's easier to think about in parallel with "EnableValidation"? The fact that it's inverted is a bit confusing.

Reply:
Sounds reasonable, though we'd need to ensure the default (true) is set consistently to preserve existing behavior.

Reply:
That looks good for me. Made changes.